app/src/main/java/com/example/teumteum/ui/friend/FriendMatchingPreviewDialog.kt

```
@@ -141,18 +164,13 @@ class FriendMatchingPreviewDialog : DialogFragment() {
-        binding.imgTeum.setImageResource(imageList[currentIndex])
-        viewModel.setTeumRequestGraphicId(currentIndex)
-    }
-
     private fun formatDate(date: String?): String {
         if (date.isNullOrBlank()) return ""
         return try {
-            val parsed = java.time.LocalDate.parse(date) // "2025-08-20"
-            parsed.format(java.time.format.DateTimeFormatter.ofPattern("yy.MM.dd")) // "25.08.20"
+            val parsed = java.time.LocalDate.parse(date)
+            parsed.format(java.time.format.DateTimeFormatter.ofPattern("yy.MM.dd"))
         } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)